packages/utils/popup-manager.ts

```diff
@@ -216,7 +211,7 @@ const getTopPopup = function () {
 
 if (!isServer) {
   // handle `esc` key when the popup is shown
-  on(window, 'keydown', function (event: KeyboardEvent) {
+  window.addEventListener('keydown', function (event: KeyboardEvent) {
```

Comment:
**Question:**  Why?

Reply:
Fix ts error

Reply:
May be we should consider enhance the typing for `on` method, instead of use `window.addEventListener` as alternative.

Reply:
I don't see any benefits of using `on` and `off`. Why do we need this function? 
Also, I plan to remove `on` and `off` in future PRs.

Reply:
`on` and `off` is a shortcut. So we have a extra file `utils/dom.ts`.

After reviewing the current code, i think we can use `addEventListener` by ourselves. This reduces both additional file maintenance and better types of readability.